src/main/java/radiounit/ManagedRadioUnitRegistry.java

```diff
@@ -0,0 +1,64 @@
+package radiounit;
+
+import java.util.List;
+
+import common.AlarmStatusLevel;
+import common.RatType;
+import common.Vendor;
+
+/**
+ * Radio Unit Registry to keep track of all Radio Units
+ * 
+ * 
+ * @author esiumat
+ *
+ */
+public class ManagedRadioUnitRegistry extends AbstractManagedRadioUnitRegistry {
```

Comment:
We'll need to add a list (or map?) or radios here

Reply:
Pardon, I'm not super clear on why. Wouldn't this have the list of radios from the abstract class it's extending?